### ISSUE TITLE
Implement home page design

### ISSUE THREAD

**Issue by rviscomi** [HTTPArchive/almanac.httparchive.org#106]
@HTTPArchive/developers 

Using the design in #46, implement the HTML/CSS/JS. For any assets that we don't have (specific fonts or images) use placeholders for now.

**Comment by tanhengyeow** [comment]
@rviscomi I'm interested in this 😄 Do we want to go with mobile-first or desktop-first implementation?

**Comment by rviscomi** [comment]
Desktop first, just because we're still waiting on the mobile designs 😁

**Comment by tanhengyeow** [comment]
@rviscomi Do we want to consider using [normalize.css](https://necolas.github.io/normalize.css/) or is custom reset preferred?

**Comment by rviscomi** [comment]
Yeah normalize SGTM 👍

**Comment by tanhengyeow** [comment]
@rviscomi Are the mobile designs/assets needed for the home page finalized? If so I can factor that in the current PR as well. 

Another approach would be that I finish up the current PR taking care of the desktop design first and follow up the mobile design in a separate PR :)

**Comment by rviscomi** [comment]
Yes, the mobile home page design is final and ready to be implemented. I think it'd make sense to do both desktop/mobile responsively in one PR since they will share the same code. Does that sound good?

**Comment by tanhengyeow** [comment]
Yep sounds great since they are all finalized 😄

**Comment by rviscomi** [comment]
Here's the link to the interactive designs with colors/fonts/etc annotated: https://projects.invisionapp.com/d/main?origin=v7#/console/18174420/379965024/inspect

**Comment by rviscomi** [comment]
@HTTPArchive/developers: is anyone available to help finish the implementation of the home page design? @tanhengyeow and I have made some progress on this in the [design branch](https://github.com/HTTPArchive/almanac.httparchive.org/tree/design). It's not quite fully responsive and there are some graphical elements still to be implemented, like the header background (see the [original designs](https://projects.invisionapp.com/d/main?origin=v7#/console/18174420/379965024/inspect)). If you're able to contribute, feel free submit a PR into the branch.

The goal is to have two stylesheets: one specifically for the home page and another more general stylesheet that is for the site-wide design.

**Comment by MSakamaki** [comment]
I can will help.
python is a beginner, javascript/css/html is good.

I will try to find implementation I can do it.

**Comment by MSakamaki** [comment]
@rviscomi 
sorry.
The link of the above link (original design) does not seem to be seen due to insufficient authority.
How can I view it?

**Comment by rviscomi** [comment]
@MSakamaki what's your email address? I'll invite you to the Invision project.

**Comment by MSakamaki** [comment]
@rviscomi 
Thank you! My email address is: mail@m-sakamaki.jp

**Comment by MSakamaki** [comment]
Is there any discussion or answer to the breakpoint (responsive width) of this website?
Currently, 600, 800, 900, and 2048 are set.
(2048px was added by me)

If not decided, is it worth discussing?
(Creating common breakpoints helps you create a general style sheet.)

**Comment by rviscomi** [comment]
No strong feelings on breakpoints. Use whatever is needed to keep the layout looking good.

**Comment by rviscomi** [comment]
@MSakamaki @tanhengyeow how are you both feeling about the home page? The `design` branch has the latest implementation. If that looks good we can merge that into master and close this issue. We can always iterate on it with small fixes as needed but I'm mostly concerned with getting the major pieces built.

**Comment by rviscomi** [comment]
Thanks @MSakamaki! I'll close this and file a new issue to resolve the browser discrepancy.

**Comment by MSakamaki** [comment]
@rviscomi @tanhengyeow 
The solution seems very straightforward.
I feel that this large 91 is a very good layout, so I agree with the idea of removing "people".
